magento/magento2
# Fix: paypal is not giving lastname in express checkout

### Description (*)
Fixed Express checkout to show lastname in the the db for billing and shipping in guest mode. Two files were changed in the Magento\Paypal\Model, Checkout.php and Api\Nvp.php. For Checkout.php, had to comment out the lines that were setting the sipping details to NULL. In Api\Nvp.php , had to change the SHIPTONAME to split up the first and last name so each field can be saved respectively instead of the whole name in firstname.


### Fixed Issues (if relevant)

1. magento/magento2#23061: paypal is not giving lastname in express checkout

### Manual testing scenarios (*)

1. Place a order on magento using express checkout in guest mode
2. After successful order, check sales_order and  sales_order_adress table in db to confirm that order information is correct.




### Contribution checklist (*)
